apps/console/src/features/claims/pages/local-claims.tsx

```diff
@@ -302,7 +303,16 @@ const LocalClaimsPage: FunctionComponent<LocalClaimsPageInterface> = (
                 }
                 isLoading={ isLoading }
                 title={ t("console:manage.features.claims.local.pageLayout.local.title") }
-                description={ t(attributeConfig.attributes.description) }
+                description={ 
+                    <p>
```

Comment:
Do we need `<p>` tags here? Check other places as well.
```suggestion
                    <p>
```

Reply:
Fixed with 249e320604fcef016aaffc5e2bb956c0cd74921f